bindings/src/main/java/io/pravega/storage/gcp/GCPChunkStorage.java

```diff
@@ -0,0 +1,221 @@
+        try (ReadChannel readChannel = this.storage.reader(this.config.getBucket(), getObjectPath(handle.getChunkName()))) {
+            try {
+                readChannel.seek(fromOffset);
+                readChannel.limit(fromOffset + length);
+                int val = readChannel.read(ByteBuffer.wrap(buffer, bufferOffset, length));
+                if (val == -1) {
+                    ChunkInfo info = doGetInfo(handle.getChunkName());
+                    if (fromOffset >= info.getLength() || (length + fromOffset) >= info.getLength()) {
+                        throw new IllegalArgumentException(handle.getChunkName());
+                    }
+                }
+                return val;
+            } catch (Exception e) {
+                throw convertException(handle.getChunkName(), "doRead", e);
+            }
+        }
+    }
+
+    @Override
+    protected int doWrite(ChunkHandle handle, long offset, int length, InputStream data) {
+        throw new UnsupportedOperationException("GCPChunkStorage does not support writing to already existing objects.");
+    }
+
+    @Override
+    public int doConcat(ConcatArgument[] chunks) throws UnsupportedOperationException {
+        throw new UnsupportedOperationException("GCP chunk storage does not support doConcat");
+    }
+
+    @Override
+    protected void doSetReadOnly(ChunkHandle handle, boolean isReadOnly) throws UnsupportedOperationException {
+        throw new UnsupportedOperationException("GCP chunk storage does not support doSetReadOnly");
+    }
+
+    @Override
+    protected ChunkInfo doGetInfo(String chunkName) throws ChunkNotFoundException {
+        Blob blob = this.storage.get(this.config.getBucket(), getObjectPath(chunkName), Storage.BlobGetOption.fields(Storage.BlobField.SIZE));
+        if (null == blob) {
+            throw new ChunkNotFoundException(chunkName, null, null);
+        }
+        return ChunkInfo.builder()
+                .name(chunkName)
+                .length(blob.getSize())
+                .build();
+    }
+
+    @Override
+    protected ChunkHandle doCreate(String chunkName) {
+        throw new UnsupportedOperationException("GCPChunkStorage does not support creating object without content.");
+    }
+
+    @Override
+    protected ChunkHandle doCreateWithContent(String chunkName, int length, InputStream data) throws ChunkStorageException {
+        BlobId blobId = BlobId.of(this.config.getBucket(), getObjectPath(chunkName));
+        BlobInfo blobInfo = BlobInfo.newBuilder(blobId).setContentType("application/octet-stream").build();
+        try (InputStream inputStream = data) {
+            this.storage.createFrom(blobInfo, inputStream);
+            return ChunkHandle.writeHandle(chunkName);
+        } catch (IOException e) {
+            throw convertException(chunkName, "doCreateWithContent", e);
+        } catch (Exception e) {
+            e.printStackTrace();
+            throw new ChunkStorageException(e.getMessage(), null, null);
```

Comment:
call `throw convertException(chunkName, "doCreateWithContent", e);`

Reply:
I have already made this change. I don't know why its refreshing. Please checks in files changed there it is showing updated code. 
![image](https://user-images.githubusercontent.com/104156566/175808170-d7c9b1f5-779f-409f-9068-08b8c86066d7.png)